src/main/java/org/folio/circulation/domain/OverduePeriodCalculatorService.java

```diff
@@ -153,4 +155,7 @@ private int getGracePeriodMinutes(Loan loan) {
     return loan.getLoanPolicy().getGracePeriod().toMinutes();
   }
 
+  private UUID getOwnerServicePoint(Loan loan) {
```

Comment:
Maybe this method should be renamed to something like getItemLocationPrimaryServicePoint? The fact that it's the same service point that we use for determining owner is probably not a coincidence, but this class knows nothing about fee/fine owners.

Reply:
@alexanderkurash, 
Thanks, renamed